eq-author-api/utils/migrations/createMigration.js

```diff
@@ -0,0 +1,21 @@
+const fs = require("fs").promises;
+
+if (process.argv.length < 3) {
+  /* eslint-disable-next-line no-console */
+  console.info(
+    "\x1b[33m",
```

Comment:
Could we use [chalk[(https://www.npmjs.com/package/chalk) instead

Reply:
I considered this but refrained from adding another dependancy. I'm happy to add Chalk if others are in agreement.

Reply:
As it won't be bundled I don't think its a big deal and I personally have no idea what that escape sequence does.